Added Point Of Sale Enum.

mnouwens opened the pull request:
I Added pointofsale as enum. This was missing in the api.

zwaldeck commented:
Can you update it to version 4.0.2 and also in the README.md

mnouwens commented:
Will doe was also adding Terminal api code.

But when running test got:

Error response from mollie with status code '403' and body:
{"status":403,"title":"Forbidden","detail":"You are not allowed to use the
Terminals API.","_links":{"documentation":{"href":"
https://docs.mollie.com/overview/handling-errors","type":"text/html"}}}

Leave the test out?

Marteijn

Op wo 4 okt 2023 om 18:24 schreef Wout Schoovaerts ***@***.***
>:

> Can you update it to version 4.0.2 and also in the README.md
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zwaldeck/mollie/pull/99#issuecomment-1747246842>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABSL4NILCGA6DZE7H5DVB73X5WETRAVCNFSM6AAAAAA5KU6AFKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTONBXGI2DMOBUGI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
With regards,

Marteijn Nouwens
Tel. 06 282 74 278

mnouwens commented:
Hoi, I also added the Terminal Api.

Please check and provide feedback

Marteijn

Op wo 4 okt 2023 om 20:06 schreef Marteijn Nouwens ***@***.***
>:

> Will doe was also adding Terminal api code.
>
> But when running test got:
>
> Error response from mollie with status code '403' and body:
> {"status":403,"title":"Forbidden","detail":"You are not allowed to use the
> Terminals API.","_links":{"documentation":{"href":"
> https://docs.mollie.com/overview/handling-errors","type":"text/html"}}}
>
> Leave the test out?
>
> Marteijn
>
> Op wo 4 okt 2023 om 18:24 schreef Wout Schoovaerts <
> ***@***.***>:
>
>> Can you update it to version 4.0.2 and also in the README.md
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/zwaldeck/mollie/pull/99#issuecomment-1747246842>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ABSL4NILCGA6DZE7H5DVB73X5WETRAVCNFSM6AAAAAA5KU6AFKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTONBXGI2DMOBUGI>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
> With regards,
>
> Marteijn Nouwens
> Tel. 06 282 74 278
>


-- 
With regards,

Marteijn Nouwens
Tel. 06 282 74 278

zwaldeck commented on src/main/java/be/woutschoovaerts/mollie/handler/terminals/TerminalHandler.java:
```diff
@@ -0,0 +1,67 @@
+ * @author Marteijn Nouwens
+ */
+@RequiredArgsConstructor
+public class TerminalHandler {
```
All looks good, But I still miss unit tests

zwaldeck commented:
No that you also added the endpoints for the POS you can bump the version also to 4.1.0 as this is a new feature added

mnouwens commented:
Fixed!

Op vr 13 okt 2023 om 17:52 schreef Wout Schoovaerts <
***@***.***>:

> No that you also added the endpoints for the POS you can bump the version
> also to 4.1.0 as this is a new feature added
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zwaldeck/mollie/pull/99#issuecomment-1761739302>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABSL4NLQZPCWZXR5VPKHTI3X7FPSDANCNFSM6AAAAAA5KU6AFI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
With regards,

Marteijn Nouwens
Tel. 06 282 74 278
